### PR TITLE
Remove unused function and correct PHPDoc

### PR DESCRIPTION
**Pull request by LukasReschke**
See https://github.com/search?q=user%3Aowncloud+registerSettings&type=Code&utf8=%E2%9C%93

### CODE REVIEW

**Comment by LukasReschke** [comment]
@owncloud-bot Retest this please.

File: lib/private/app.php [hunk 1]
```diff
@@ -720,13 +721,6 @@ public static function getForms($type) {
 	}
 
 	/**
-	 * register a settings form to be shown
-	 */
-	public static function registerSettings($app, $page) {
-		self::$settingsForms[] = $app . '/' . $page . '.php';
```
Comment (DeepDiver1975) [inline lib/private/app.php]: wow - nice - feel free to kill $settingsForms as well - THX

**Comment by DeepDiver1975** [comment]
@karlitschek looks like settings is some left over from previous times - you remember the reason why this is kind of abandoned today? THX

File: lib/private/app.php (hunk 1 above)
Comment (MorrisJobke) [inline lib/private/app.php]: Kill the `settings/settings.php` too ... because there is no route that calls this file.

Comment (MorrisJobke) [inline lib/private/app.php]: There is a route, but it's an empty page ;)

Comment (MorrisJobke) [inline lib/private/app.php]: Even back in stable5 it wasn't filled with anything.

Comment (LukasReschke) [inline lib/private/app.php]: Done. I also like `// This is private as well. It simply works, so don't ask for more details` … :-/

**Comment by DeepDiver1975** [comment]
:+1:

**Comment by DeepDiver1975** [comment]
looks like really outdated code .... kill it with :fire:

**Comment by LukasReschke** [comment]
@owncloud-bot Retest this please

**Comment by MorrisJobke** [comment]
Still works :+1:

**Pull request merged**